.gitignore

```diff
@@ -6,4 +6,5 @@ yarn.lock
 .env
 .vscode
 *.rdb
-process.yml
\ No newline at end of file
+process.yml
+build
```

Comment:
Going forward since we'll be deploying often, we need to leave build out of the git ignore.